### ISSUE TITLE
Buffer overrun in panic()

### ISSUE THREAD

**Issue by gdamore** [nanomsg/nng#443]
Switching to gcc-8 found a bug, which may have been responsible for corrupted output on test failures:

```
../src/core/panic.h: In function 'nni_panic':
../src/core/panic.c:59:44: warning: '%s' directive output may be truncated writing up to 127 bytes into a region of size 121 [-Wformat-truncation=]
  (void) snprintf(buf, sizeof(buf), "panic: %s", fbuf);
                                            ^~   ~~~~
../src/core/panic.c:59:9: note: 'snprintf' output between 8 and 135 bytes into a destination of size 128
  (void) snprintf(buf, sizeof(buf), "panic: %s", fbuf);
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by gdamore** [comment]
Actually, re-reviewing, it's ok.  Truncation is expected here, but I'll fix it anyway.

**Comment by gdamore** [comment]
Fixed in ec84c5a8406bb203d5f8830e280f93cf1f63cd6a
